<launch>
  <arg name="image_raw_topic" default="/ackermann_vehicle/camera2/rgb/image_raw"/>
  <arg name="twist_cmd_topic" default="/vel_cmd"/>
  <arg name="twist_linear_x" default="1"/>

    <node name="signal_panel_recognition" pkg="signal_recognition" type="signal_recognition.py" output="screen">
      <param name="image_raw_topic" value="$(arg image_raw_topic)"/>
      <param name="twist_cmd_topic" value="$(arg twist_cmd_topic)"/>
      <param name="twist_linear_x" value="$(arg twist_linear_x)"/>
    </node>

</launch>